### DISCUSSION TITLE
Where is SMPlayer saving Playlists

### DISCUSSION THREAD

**Discussion by hollysheep** [smplayer-dev/smplayer#1067] [Q&A]
I followed this http://old-forum.smplayer.info/viewtopic.php?f=2&t=12540 to be able to jump to the next video file in the folder. I don't create a playlist as a real file with "save as". But there must a a location where smplayer ist saving this automatically created playlist. where is it? i use ubuntu 22.04 and can't find it in ~/.config, ~/.cache or ~/.locale/share.

**Comment by smplayer-dev** [comment]
`~/.config/smplayer/playlist.ini`
